Helper/Data.php

```diff
@@ -76,6 +77,16 @@ public function getDSN()
         return $this->config['dsn'];
     }
 
+    public function getIgnoreJsErrors(): array
+    {
+        $list = $this->config['ignore_js_errors'];
+        if ($list && !is_array($list)) {
```

Comment:
This check will not throw an exception if `$list` is `false` however `$list ?? []` will check for `null`
So `getIgnoreJsErrors` would return false instead of an array.

I'd suggest turning this into a 
```suggestion
        if (isset($list) && !is_array($list)) {
```

Reply:
i changed the code a little, so it should fit your needs.